lib/components/calendar/event_list.dart

```diff
@@ -4,7 +4,7 @@ import '../../model/event.dart';
 import 'event_card.dart';
 
 class EventList extends StatelessWidget {
-  const EventList({Key? key, required this.eventList}) : super(key: key);
+  const EventList({super.key, required this.eventList});
```

Comment:
, po eventList

Reply:
hej, zapomniałeś chyba o czymś 
![the-shrek-menagerie-v0-252g70camryb1 (1)](https://github.com/Bartosiki/DW-Flutter-App/assets/71559210/2b74b233-8b53-49f5-b052-e477dcbc4bd2)